src/ui.h

```diff
@@ -691,7 +689,6 @@ class GraphicsWindow {
     void RemoveConstraintsForPointBeingDeleted(hEntity hpt);
     void FixConstraintsForRequestBeingDeleted(hRequest hr);
     void FixConstraintsForPointBeingDeleted(hEntity hpt);
-    void EditConstraint(hConstraint constraint);
```

Comment:
Not sure why this got removed.. I'm hitting a build issue related to it. Perhaps this was an accidental change?